src/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  font-family: Arial;
+  font-size: 13px;
+}
+
+h1 {
+  font-size: 17px;
+  font-weight: bold;
+  margin: 0;
+}
+
+h2 {
+  font-size: 15px;
+  padding: 1em 0;
```

Comment:
this is more like margins than padding since you need an indentation from the nearby container and not inside your container